ts/screens/wallet/WalletsScreen.tsx

```diff
@@ -84,9 +85,11 @@ class WalletsScreen extends React.Component<Props> {
         wallet={item}
         isFavorite={isFavorite}
         onSetFavorite={(willBeFavorite: boolean) =>
-          this.props.setFavoriteWallet(
-            willBeFavorite ? item.idWallet : undefined
-          )
+          willBeFavorite
+            ? this.props.setFavoriteWallet(item.idWallet)
+            : this.props.wallets.length > 0
```

Comment:
```suggestion
            : this.props.wallets.length > 1
```
Should we show an alert only if the user set another wallet as favourite?

Reply:
PS: last commit on [io-dev-server ](https://github.com/pagopa/io-dev-api-server)now supports set wallet as favourite

Reply:
@Undermaken If we adopt this condition we should avoid the click on the star icon when we have only one method available.
At the moment we show the alert anytime the user's trying to remove a favourite method